apps/studio/src/lib/db/authentication/azure.ts

```diff
@@ -153,6 +171,31 @@ export class AzureAuthService {
     }
   }
 
+  private async getAzureCLIToken(): Promise<AuthConfig | null> {
+    return new Promise((resolve, reject) => {
+      const command = "az account get-access-token --resource https://ossrdbms-aad.database.windows.net --output json";
+      exec(command, { encoding: "utf8" }, (error, stdout) => {
+        if (error) {
+          console.error("Error getting token:", error);
+          reject(null);
```

Comment:
why reject(null)? Shouldn't we reject with the error?

Also -- you need to use @bksLogger, vs console for logging